doc/user-guide.md

```diff
@@ -74,7 +74,7 @@ Each `RuntimeComponent` CR must at least specify the `applicationImage` paramete
 | `service.consumes` | An array consisting of services to be consumed by the `RuntimeComponent`. |
 | `service.consumes[].category` | The type of service binding to be consumed. At this time, the only allowed value is `openapi`. |
 | `service.consumes[].name` | The name of the service to be consumed. If binding to a `RuntimeComponent`, then this would be the provider's CR name. |
-| `service.consumes[].namespace` | The namespace of the service to be consumed. If binding to a `RuntimeComponent`, then this would be the provider's CR name. ||
+| `service.consumes[].namespace` | The namespace of the service to be consumed. If binding to a `RuntimeComponent`, then this would be the provider's CR namespace. ||
```

Comment:
We've added an extra `|` at the end of the line. It shows up fine in the rendered page